temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // The common_name in the certificate
+    string common_name = 1;
+    // The organization in the certificate
+    string organization = 2;
+    // The organizational_unit in the certificate
+    string organizational_unit = 3;
+    // The subject_alternative_name in the certificate
+    string subject_alternative_name = 4;
+}
+
+message CodecServerPropertySpec {
+    // Server endpoints
+    string endpoint = 1;
+    // Whether to pass access token, i.e. jwt
+    bool pass_access_token = 2;
+    // Whether to include credentials
+    bool include_credentials = 3;
+}
+
+message NamespaceSpec {
```

Comment:
please include cost_center as we are adding this with the cost center per namespace initiative now
type string
optional

comment - optionally tag a namespace with a cost center. Adding a cost center to a namespace allows for cost center aggregations to appear on an invoice and in usage metrics. This is useful if you want to attribute Namespace usage and costs to a specific cost center.

Reply:
Please include Export configuration as a part of this spec or in the Namespace message as a separate message type